tips/TIP-0038/tip-0038.md

```diff
@@ -0,0 +1,1071 @@
+      </details>
+      <details>
+        <summary>Restricted Address</summary>
+        <blockquote>An address that contains another address and allows for configuring its capabilities. Defined in <a href='../TIP-0050/tip-0050.md#restricted-address'>TIP-50 (Restricted Address)</a>.</blockquote>
+      </details>
+    </td>
+  </tr>
+</table>
+
+Whenever a chain account mints an NFT on layer 1 on behalf of some user, the `Issuer` field can only contain the chain's
+address, since the user does not sign the layer 1 transaction. As a consequence, artists would have to mint NFTs
+themselves on layer 1 and then deposit it to chains if they want to place their own address in the `Issuer` field.
+
+### Metadata Feature
+
+Outputs may carry additional data with them that is interpreted by higher-layer applications built on the Tangle. The
+protocol treats this metadata as pure binary data, it has no effect on the validity of an output except that it
+increases the required storage deposit. ISC is a great example of a higher-layer protocol that makes use of <i>Metadata
+Feature</i>: smart contract request parameters are encoded in the metadata field of outputs.
+
+<details>
+  <summary>Metadata Feature</summary>
+  <blockquote>Defines a map of key-value pairs that is stored in the output.</blockquote>
+</details>
+<table>
+  <tr>
+    <td>
+      <b>Name</b>
+    </td>
+    <td>
+      <b>Type</b>
+    </td>
+    <td>
+      <b>Description</b>
+    </td>
+  </tr>
+  <tr>
+    <td>Feature Type</td>
+    <td>uint8</td>
+    <td>Set to <strong>value 2</strong> to denote a <i>Metadata Feature</i>.</td>
+  </tr>
+  <tr>
+    <td>Entries Count</td>
+    <td>uint8</td>
+    <td>The number of entries in the map.</td>
+  </tr>
+  <tr>
+    <td valign="top">Entries <code>anyOf</code></td>
+    <td colspan="2">
+      <details>
+        <summary>Metadata Entry</summary>
+        <blockquote>A map entry consisting of a string key and an arbitrary byte value.</blockquote>
+        <table>
+          <tr>
+            <td>
+              <b>Name</b>
+            </td>
+            <td>
+              <b>Type</b>
+            </td>
+            <td>
+              <b>Description</b>
+            </td>
+          </tr>
+          <tr>
+            <td>Key</td>
+            <td>(uint8)ByteArray</td>
+            <td>A string which may only consist of printable ASCII characters. A leading uint8 denotes its length.</td>
+          </tr>
+          <tr>
+            <td>Value</td>
+            <td>(uint16)ByteArray</td>
+            <td>An array of arbitrary binary data. A leading uint16 denotes its length.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+</table>
+
+#### Additional Transaction Syntactic Validation Rules
```

Comment:
Yes, I think so, since e.g. Transaction and Block IDs depend on the serialization. Hence, all implementations should produce the same serialization of a Transaction and hence features.